### ISSUE TITLE
Can't subscribe to some forums despite Tapatalk support

### ISSUE THREAD

**Issue by lemtom** [vranki/siilihai-client#9]
Several forums I've tried that don't work:
* https://forum.rebelscum.com/
* https://www.9lives.be/forum/
* https://www.cafemickey.com/
* https://www.spaargids.be/forum/

Is there anything I'm misunderstanding?

**Comment by vranki** [comment]
Looks like Tapatalk has changed their API and "Tapatalk technical support to third party implementations is not available at this time.". Perhaps they don't want third party apps to work anymore. 

A truly open standard for forum access would be great. I don't know if there is anything that can be done to fix this issue. I'd recommend using Discourse forum as it has a API and no Tapatalk needed.

**Comment by lemtom** [comment]
The Tapatalk-using forums that do work are on an older version of Tapatalk, then?

Very unfortunate.

**Comment by vranki** [comment]
Yep.. They have endpoint at /mobiquo/mobiquo.php. On the broken forums it doesn't exist and I don't know where it should be and what is the new API. Unfortunate indeed, but this is what you get with "standards" made by one company...
